pkg/task/tasklastrun.go

```diff
@@ -65,14 +64,3 @@ func FilterByRef(taskruns []v1beta1.TaskRun, kind string) []v1beta1.TaskRun {
 	}
 	return filtered
 }
-
-// this will filter the taskkrun which have reference to Task
```

Comment:
This isn't used anywhere in the CLI, so I removed it. Let me know if there are considerations around it I am not aware of.

Reply:
I am also not sure about this.

Reply:
/cc @pradeepitm12 Not sure if you are more familiar with this?